test/t8_IO/t8_gtest_vtk_reader.cxx

```diff
@@ -32,10 +32,20 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 TEST (t8_cmesh_vtk_reader, dummy_test)
 {
 #if T8_WITH_VTK
-  /* Currently the function crashes if compiled with vtk. */
-#else
   t8_cmesh_t          cmesh =
-    t8_cmesh_vtk_reader (NULL, 0, 0, sc_MPI_COMM_NULL, VTK_FILE_ERROR);
+    t8_cmesh_vtk_reader ("non-existing-file.vtu", 0, 0, sc_MPI_COMM_WORLD,
```

Comment:
Why not tested against an existing file?

Reply:
Because in that stage of the development we do not translate the vtkDataSet into a cmesh yet, and the function is incomplete yet.

Reply:
Alright.